tests/fault_tolerance/cancellation/test_vllm.py

```
@@ -334,6 +334,23 @@ def test_request_cancellation_vllm_remote_prefill_cancel(
                     pattern="issued control message Kill to sender",
                 )
 
+                # Verify decode worker never received the request
+                pattern = "Request ID: "
+                try:
+                    _, decode_log_offset = poll_for_pattern(
```

Ruff (0.14.5)

Unpacked variable `decode_log_offset` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)